### ISSUE TITLE
Safari Content Blockers crash if the blocker list is too large

### ISSUE THREAD

**Issue by saagarjha** [saagarjha/OpenAdBlock#3]
Seems to be a bug in the content blocker loader. Anyone not experiencing this problem?

**Comment by azinicus** [comment]
I think this is the same issue being experienced by similar project https://github.com/rocketshipapps/adblockfast/issues/26

**Comment by saagarjha** [comment]
Yes it looks like it. I'm not sure if it's a bug with the CSS or `SFContentBlockerManager`, since all I get is a crash log.

**Comment by azinicus** [comment]
Although it seems from my brief testing that the .json from this project actually blocks things whereas the AdblockFast one does not currently initiate by iOS (that one is much larger btw)

**Comment by saagarjha** [comment]
This was the most complex one that seemed to work.

**Comment by oldestlivingboy** [comment]
Safari is meant to handle up to 50K rules, so your list is unlikely too big. In the case of Adblock Fast, I introduced a syntax error that Safari was failing silently on.

**Comment by saagarjha** [comment]
That might be it. I'll have to look into it.

**Comment by mcfedr** [comment]
Interested where the number 50K comes from

**Comment by saagarjha** [comment]
I believe this number has been increased recently, actually. I'll look into it.
